### PR TITLE
Feature/clean button

### PR DESCRIPTION
**Pull request by mikibonacci**
This fixes https://github.com/aiidalab/aiidalab-qe/issues/815

For now, the `clean remote data` button is always shown for a finished workchain; if the remote folders are empty, the button is still there but is disabled. I am not sure if we should always display it, and that the name of the button is the best suited one. 

waiting for some comments @AndresOrtegaGuerrero , @superstar54

### CODE REVIEW

File: src/aiidalab_qe/app/result/__init__.py [hunk 1]
```diff
@@ -120,6 +138,32 @@ def _update_kill_button_layout(self):
         else:
             self.kill_button.disabled = True
 
+    def _update_clean_scratch_button_layout(self):
+        """Update the layout of the kill button."""
+        # The button is hidden by default, but if we load a new process, we hide again.
+        if self.process is None or self.process == "":
```
Comment (AndresOrtegaGuerrero) [inline src/aiidalab_qe/app/result/__init__.py:144]: `if not self.process:`

File: src/aiidalab_qe/app/result/__init__.py [hunk 2]
```diff
@@ -120,6 +138,32 @@ def _update_kill_button_layout(self):
...
+            cleaned_bool = []
+            for called_descendant in process.called_descendants:
+                if isinstance(called_descendant, orm.CalcJobNode):
+                    try:
+                        cleaned_bool.append(
+                            called_descendant.outputs.remote_folder.is_cleaned
+                        )
+                    except (OSError, KeyError):
+                        pass
+            self.clean_scratch_button.disabled = False not in cleaned_bool
```
Comment (AndresOrtegaGuerrero) [inline src/aiidalab_qe/app/result/__init__.py:165]: `not any(clean_bool)`
Does this one work ?

Comment (mikibonacci) [reply src/aiidalab_qe/app/result/__init__.py:165]: no, but `self...disabled=all(cleaned_bool)` will

Comment (mikibonacci) [reply src/aiidalab_qe/app/result/__init__.py]: I implemented it

File: src/aiidalab_qe/app/result/__init__.py (hunk 1 above)
Comment (mikibonacci) [reply src/aiidalab_qe/app/result/__init__.py]: Done!

**Review by AndresOrtegaGuerrero — APPROVED** [review]
LGTM! , thank you Miki

**Pull request merged**